### ISSUE TITLE
Word Language Model

### ISSUE THREAD

**Issue by siddsach** [pytorch/examples#256]
I noticed in the word language model example, no activation function is used. It's just a linear layer to get predictions, without the softmax that usually would be there? Was wondering why?

**Comment by soumith** [comment]
the loss function includes the logsoftmax

**Comment by siddsach** [comment]
Oh lol okay, my bad.
